health/health.d/hdfs.conf

```diff
@@ -28,7 +30,10 @@ component: HDFS
     every: 10s
      warn: $this > 0
     delay: down 15m multiplier 1.5 max 1h
-     info: number of missing blocks
+     info: Number of missing blocks. \
+           There are missing blocks on some datanodes. \
+           It may indicate a problem with the underlying storage or filesystem of a datanode. To fix, you can find what blocks are missing using hdfs fsck and bring the blocks back online. \
```

Comment:
This line is long when we compare with other alarms, is this expected?

Reply:
Indeed, a nice catch, @thiagoftsm :hawkeye: